webapp/channels/src/components/post_markdown/system_message_helpers.tsx

```diff
@@ -195,8 +195,16 @@ function renderHeaderChangeMessage(post: Post): ReactNode {
     };
 
     const username = renderUsername(post.props.username);
-    const oldHeader = post.props.old_header ? renderFormattedText(post.props.old_header, headerOptions, post) : null;
-    const newHeader = post.props.new_header ? renderFormattedText(post.props.new_header, headerOptions, post) : null;
+    const oldHeader = post.props.old_header ? renderFormattedText(
+        post.props.old_header.replace(/\n/g, '\n\n'),
+        headerOptions,
+        post,
+    ) : null;
+    const newHeader = post.props.new_header ? renderFormattedText(
+        post.props.new_header.replace(/\n/g, '\n\n'),
```

Comment:
Why is this needed? wouldn't removing the `singleline` from the options on line 192 solve the issue?

Reply:
Yes, you're right. Simply removing the single line from line 192 resolves the matter.